sprint/prep-work.md

```diff
@@ -0,0 +1,35 @@
+# Sprint preparation
+
+## Checklist
+
+:::{list-table}
+:header-rows: 1
+:stub-columns: 1
+:widths: 1 4 1 2
+
+* - #
+  - Task
+  - Status
+  - Notes
+* - 1
+  - Have a GitHub account?
+  - {fas}`check;sd-text-success`
+  - 
+
+:::
+
+
+## Sprint day
+
+%TODO: Improve text%
+
+:::{tip}
+To communicate during the sprint, we will be using Discord. The [Using the Discord Server](https://youtu.be/w2A8SknM-68) introduction video contains a brief introduction to the Discord Data Umbrella server with some tips on how to navigate the application.
+:::
+
+### Pair programming
+
+Live Share is an extension from Visual Studio Code that allows you to collaboratively edit and debug with others in real-time. It allows you to instantly share your current project, edit snippets of code at the same time or follow someone’s cursor while they program. To learn more about Live Share, visit [Live Share with Visual Studio Code](https://youtu.be/WHOiljOYGVw).
+
```

Comment:
Can we embed the video here?
Can we also include a note that the video, I think, is about 15 minutes long.

Reply:
Done